### PR TITLE
Product & profile page

### PR DESCRIPTION
**Pull request by kshitijthatte**
- added user profile page ( to view, click on 'Hi, User' on the navbar)
- added a single product page
- added cart checkout page ( to view, after adding items to the cart, click 'place order')

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ import SignupPage from "./pages/SignupPage";
 import Wishlist from "./pages/Wishlist";
 import PrivateRoute from "./components/PrivateRoute";
 import Cart from "./pages/Cart";
+import UserProfilePage from "./pages/UserProfilePage";
+import SingleProductPage from "./pages/SingleProductPage";
+import CheckoutPage from "./pages/CheckoutPage";
```
Comment (someshtiwarii) [inline src/App.js:14]: can use barrel export

**Review by someshtiwarii — APPROVED** [review]
clean code! good ot merge

File: src/App.js (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply src/App.js:14]: can 
use barrel exports

**Review by Shrinkhlarajpoot — APPROVED** [review]
Clean and structured code...You can merge it.

**Pull request merged**